pkg/xray/provider.go

```diff
@@ -157,22 +157,12 @@ func (p *XrayProvider) Configure(ctx context.Context, req provider.ConfigureRequ
 		)
 	}
 
-	version, err := util.GetXrayVersion(restyClient)
-	if err != nil {
-		resp.Diagnostics.AddError(
-			"Error getting Xray version",
-			err.Error(),
-		)
-		return
-	}
-
 	featureUsage := fmt.Sprintf("Terraform/%s", req.TerraformVersion)
 	go util.SendUsage(ctx, restyClient.R(), productId, featureUsage)
```

Comment:
Maybe we could drop the telemetry too if we really need to keep this fork around. :smile:

Reply:
Ah yeah. This should be an opt-in knob. Good catch.